app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js

```diff
@@ -29,15 +29,27 @@ const getPrefKey = (application) => {
     : application.preference_record_type
 }
 
+export const getAccessibilityKeys = (application) => {
+  const accessibilityKeys = compact(Object.keys(application.has_ada_priorities_selected))
```

Comment:
Question: previously there was `... || []` to handle an empty value, is that no longer necessary?

Reply:
No longer necessary because I'm checking for existence of has_ada_priorities_selected before it enters this function now